app/controllers/admin/AdminController.php

```diff
@@ -155,9 +155,14 @@ public function applicationsShow($id)
     // would significantly reduce the number of SQL queries needed, as well
     // as the number of variables passed to the view.
     $application = Application::getUserApplication($id);
-    $profile = Profile::getUserProfile($id);
+    //only load the user profile if it has been filled out
+    if (Profile::where('user_id', '=', $id)->exists()) {
+      $profile = Profile::getUserProfile($id);
+    }
```

Comment:
They can! If you just fill in your name, email, and password and click "create account" you show up as a user but do not yet appear in the "profiles" table. Only after they fill out the "basic info" page aka where you are taken after you hit "create account" do they have a profile.